pyproject.toml

```diff
@@ -33,7 +33,9 @@ mkdocs = "^1.2.2"
 mkdocstrings = "^0.15.2"
 mkdocs-jupyter = { version = "^0.17.3", python = ">=3.7" }
 mkdocs-material = "^7.2.5"
-tensorflow-cpu = "^2.6.0"
+einops = "^0.3.2"
+torchmetrics = "^0.5.1"
+tensorflow = "^2.6.0"
```

Comment:
Can que use `tensorflow-cpu` instead? Its much lighter for CI and we don't use it for training. For `elegy` I've been using:

```toml
tf-nightly-cpu = "2.8.0.dev20211103"
```